insta/src/output.rs

```diff
@@ -356,6 +411,15 @@ fn print_info(metadata: &MetaData) {
     }
 }
 
+fn encode_file_link_escape(path: &Path) -> String {
+    assert!(path.is_absolute());
+    format!(
+        "\x1b]8;;file://{}\x1b\\{}\x1b]8;;\x1b\\",
```

Comment:
Could I ask what this does / could we add a comment?

Reply:
I've added a comment.

Reply:
Thanks + great idea re the clickable link!